build/test.sh

```diff
@@ -28,12 +72,20 @@ function run_python_tests() {
   docker run cortexlabs/test
 }
 
-cmd=${1:-""}
+function run_api_tests() {
+  if [ "$provider" = "aws" ]; then
+    pytest $ROOT/test/e2e/tests -k aws --aws-env "$cluster_env"
+  elif [ "$provider" = "gcp" ]; then
+    pytest $ROOT/test/e2e/tests -k gcp --gcp-env "$cluster_env"
+  fi
+}
```

Comment:
can we call this `run_e2e_tests` instead? for consistency

Reply:
Yep, I agree.